WordPress/Classes/Models/QuickStartTourState.swift

```diff
@@ -0,0 +1,10 @@
+@objc(QuickStartTourState)
+open class QuickStartTourState: NSManagedObject {
+    // Relations
+    @NSManaged open var blog: Blog?
+    @NSManaged open var completed: Bool
+    @NSManaged open var skipped: Bool
```

Comment:
This works but allows for the possibility for a tour to be both completed and skipped which doesn't make sense.  I'm cool if we want to just run with it (we can be careful to manage two boolean state properties) but if there was ever a third potential state I'd consider having a single `state` property backed by an enum integer or string.

Reply:
I think it does make sense for a tour to be both completed and skipped… skipped tours can still be completed by going into the checklist and tapping on it, then completing it. Skipping means it was suggested, but "Not now" was tapped. We have to remember how many were skipped because after three skips, we stop making suggestions.

Reply:
> We have to remember how many were skipped because after three skips, we stop making suggestions.

Ah! Right. It makes sense to have both then.  Thanks for clarifying :)